dcompiler.dd

```diff
@@ -54,6 +58,9 @@ $(H2 $(LNAME2 requirements, Requirements and Downloads))
 $(H2 $(LNAME2 installation, Installation))
 
     $(WINDOWS
+$(MESSAGE_BOX gray, $(B Hint) - The official $(LINK2 $(ROOT_DIR)download.sh, installer)
+performs these steps automatically.
+)
```

Comment:
This documentation is also displayed via e.g. -man and linked from the release pages, so chances are actually very high that the user has already installed DMD and just looks for a documentation of the CLI flags.

Reply:
That install script requires a bash environment, and I suspect most Windows users don't have that.